documentation/.yarnrc.yml

```
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

```suggestion
nodeLinker: node-modules

```

You can remove the `.yarn` folder now as we use the new `core pack enable` to make sure the correct version of yarn is used